lighthouse-viewer/app/src/firebase-auth.js

```diff
@@ -24,11 +24,13 @@ export class FirebaseAuth {
     this._provider.addScope('gist');
 
     firebase.initializeApp({
-      apiKey: 'AIzaSyApMz8FHTyJNqqUtA51tik5Mro8j-2qMcM',
-      authDomain: 'lighthouse-viewer.firebaseapp.com',
-      databaseURL: 'https://lighthouse-viewer.firebaseio.com',
-      storageBucket: 'lighthouse-viewer.appspot.com',
-      messagingSenderId: '962507201498',
+      apiKey: 'AIzaSyBQEZMlX6A9B0jJ6PFGcBADbXZG9ogyCmQ',
+      authDomain: 'lighthouse-chrom-1560304954232.firebaseapp.com',
```

Comment:
it's really `lighthouse-chrom` not `lighthouse-chrome`?

Reply:
Shrug. It was truncated. This is the correct credentials.